webpack.config.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable */
```

Comment:
- [ ] Kindly enable the linters here too. 🤓

Reply:
[issue resolved](https://github.com/Wahaj-Ali/tv-hub/commit/87e19a9f0c9cd7e81117ab40430afcbf2e65235b)